### PR TITLE
Additional settings mnrva 511

### PR DESCRIPTION
**Pull request by RiaRenold**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-511

 ## Description:
1. As a user navigate to the list of monitors
2. Select a monitor
3. On this monitor detail page select Additional Settings
4. Tests the slide down and that the texts appear in the correct areas.
 
## Testing:
npm run  test:e2e
 
## Screenshots:
(if applicable)

### CODE REVIEW

File: e2e/src/pages/monitordetailspage.ts [hunk 1]
```diff
@@ -3,12 +3,16 @@ import { element, by, WebElement } from "protractor"
 
 export class monitorsDetailsPage
```
Comment (pras7430) [inline e2e/src/pages/monitordetailspage.ts:4]: we are following the Pascal casing for class name

File: e2e/src/pages/resourcesdetailspage.ts [hunk 2]
```diff
@@ -3,17 +3,12 @@ import { element, by } from "protractor";
 export class resourcesDetailsPage
```
Comment (pras7430) [inline e2e/src/pages/resourcesdetailspage.ts:3]: Pascal case

File: e2e/src/pages/resourcesdetailspage.ts [hunk 3]
```diff
@@ -3,17 +3,12 @@ import { element, by } from "protractor";
...
-    confirmBtnOnDeleteResourceOverlay=element(by.xpath("//hx-modal[@id='delResModal']//footer//button[contains(text(),'Confirm')]"));
-
-    metadatapencil=element(by.xpath("//hx-disclosure[@id='metapop']"));
-    
-    metadataPopover=element(by.xpath("//hx-popover[@id='metaPopover']"));
+    actionsDropdown                    =element(by.xpath("//hx-disclosure[@class='hxBtn']//hx-icon[@class='hxPrimary']"));
+    actionDropdown_options             =element.all(by.tagName("hx-menuitem"));
+    deleteResourceOverlayHeader        =element(by.xpath("//hx-modal[@id='delResModal']//header"));
+    confirmBtnOnDeleteResourceOverlay  =element(by.xpath("//hx-modal[@id='delResModal']//footer//button[contains(text(),'Confirm')]"));
+    metadatapencil                     =element(by.xpath("//hx-disclosure[@id='metapop']"));
```
Comment (pras7430) [inline e2e/src/pages/resourcesdetailspage.ts:10]: we should have similar naming conventions (Camel casing)

File: e2e/src/pages/resourcesdetailspage.ts [hunk 4]
```diff
@@ -3,17 +3,12 @@ import { element, by } from "protractor";
...
-    
-    deleteResourceOverlayHeader=element(by.xpath("//hx-modal[@id='delResModal']//header"));
-    
-    confirmBtnOnDeleteResourceOverlay=element(by.xpath("//hx-modal[@id='delResModal']//footer//button[contains(text(),'Confirm')]"));
-
-    metadatapencil=element(by.xpath("//hx-disclosure[@id='metapop']"));
-    
-    metadataPopover=element(by.xpath("//hx-popover[@id='metaPopover']"));
+    actionsDropdown                    =element(by.xpath("//hx-disclosure[@class='hxBtn']//hx-icon[@class='hxPrimary']"));
+    actionDropdown_options             =element.all(by.tagName("hx-menuitem"));
```
Comment (pras7430) [inline e2e/src/pages/resourcesdetailspage.ts:7]: naming convention

File: e2e/src/features/monitoring/monitordetails/monitorDetailsFieldDisplay.e2e-spec.ts [hunk 5]
```diff
@@ -43,7 +40,28 @@ describe("Test label field display on Monitor's details page", ()=> {
...
       });
+   });
+
+
+      it("Verify that additional settings slide down on clicking it and that the text appears in correct areas",()=>{
+         page2.additionalSettings.click();
+         
+         expect(obj["period(seconds)"]).toEqual(66);
+         
+         expect(obj.excludedResourceIds[0]).toEqual('development:5')
```
Comment (pras7430) [inline e2e/src/features/monitoring/monitordetails/monitorDetailsFieldDisplay.e2e-spec.ts:51]: it should match with UI instead of hardcode value here

**Comment by FarajiA** [comment]
Also looks as tho you'll have to resolve your conflict to merge before merging this PR.
This will also make the build check pass.

File: src/app/_mocks/monitors/single.json [hunk 6]
```diff
@@ -5,6 +5,7 @@
     "interval": "PT1M6S",
     "resourceId": "development:2",
     "policy": false,
+    "period(seconds)":66,
```
Comment (FarajiA) [inline src/app/_mocks/monitors/single.json:8]: The parenthesis after the word 'period' is not needed.

File: e2e/src/features/monitoring/monitordetails/monitorDetailsFieldDisplay.e2e-spec.ts (hunk 5 above)
Comment (RiaRenold) [reply e2e/src/features/monitoring/monitordetails/monitorDetailsFieldDisplay.e2e-spec.ts:51]: Okay,,,will change in my next commit.

File: e2e/src/pages/monitordetailspage.ts (hunk 1 above)
Comment (RiaRenold) [reply e2e/src/pages/monitordetailspage.ts:4]: Okay.

File: e2e/src/pages/resourcesdetailspage.ts (hunk 2 above)
Comment (RiaRenold) [reply e2e/src/pages/resourcesdetailspage.ts:3]: Okay

File: e2e/src/pages/resourcesdetailspage.ts (hunk 4 above)
Comment (RiaRenold) [reply e2e/src/pages/resourcesdetailspage.ts:7]: that will be done

File: e2e/src/pages/resourcesdetailspage.ts (hunk 3 above)
Comment (RiaRenold) [reply e2e/src/pages/resourcesdetailspage.ts:10]: Okay

**Pull request merged**